test/pages/PageConstantsSpec.scala

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package pages
+
+import models.{RentARoom, Rentals, RentalsAndRentARoom}
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpec
+
+import scala.language.postfixOps
+
+class PageConstantsSpec extends AnyWordSpec with Matchers {
+
```

Comment:
Added a test here to make sure that all paths are as expected. This test will fail if someone changes the names of the json labels in the FE userAnswers. If someone needs to update the label names they will have to update this test as well.